Source/Core/VideoBackends/Software/SWOGLWindow.cpp

```diff
@@ -25,10 +27,9 @@ void SWOGLWindow::Init(void *window_handle)
 void SWOGLWindow::Shutdown()
 {
 	GLInterface->Shutdown();
-	delete GLInterface;
-	GLInterface = nullptr;
+	GLInterface.reset();
 
-	SWOGLWindow::s_instance.release();
+	s_instance.reset();
```

Comment:
Fixes a memory leak here, since releasing just gets rid of ownership of the allocated resource and doesn't deallocate it.